application/routes/user.py

```
@@ -21,14 +21,15 @@
 from .. import api, profiles
-
+import logging
+logger = logging.getLogger(__name__)
 
 # TODO: review the need to remove the argument readings from the APIs, once we finish debugging
 
 @api.route('/userid')
 def index():
     user = profiles.get_user()
-
+    logger.info(f"User: Fetching user ID for user: {user.id}")
```

Ruff

Logging statement uses f-string

(G004)